packages/x-charts-pro/src/ChartZoomSlider/internals/ChartAxisZoomSlider.tsx

```diff
@@ -141,6 +147,136 @@ export function ChartAxisZoomSlider({ axisDirection, axisId }: ChartZoomSliderPr
   );
 }
 
+interface ChartAxisZoomSliderTrackProps extends React.ComponentProps<'rect'> {
+  axisId: AxisId;
+  axisDirection: 'x' | 'y';
+  reverse: boolean;
+}
+
+function ChartAxisZoomSliderTrack({
```

Comment:
This file is getting huge. I'll move the components to other files in a follow-up.